Update `Ranking criteria` with FA tag usage and AI content crediting (#10175)
ppy/osu-wiki: wiki/Ranking_criteria/en.md

### Guidelines

- **Creators of the media used in a beatmap should be credited in the beatmap's description.** Per the [Content Usage Guidelines](/wiki/Rules/Content_usage_guidelines), media used in beatmaps should be permitted for use in osu!, and their licensing will likely detail how to credit their creators. Common reasons to omit credits from the description include exceeding difficulty in sourcing, creators' wishes to remain uncredited, and presence of credits in the artist, title, or source fields.

>   - Beatmaps using AI generated backgrounds should instead credit it as such in the description.

Reviewer: ```suggestion
  - Beatmaps using AI-generated backgrounds should instead credit them as such in the description.
```

Author: Updated.

>   - Beatmaps using AI generated backgrounds should instead credit it as such in the description.

Reviewer: would it make sense to put "media" here instead of "backgrounds"?

Author: Back in the meeting we wanted to narrow the scope to backgrounds for now. Using "media" would potentially include music as well, and we discussed that songs/maps/etc... weren't mature enough to consider as of now.

Reviewer 2: I don't get the differentiation. Please use "media" as it should t matter what the content is.

Author: It can be done, but this may cause some confusion that was previously discussed. Do AI vocals on, for example, AI covers count? In practice, we would only be making sure the backgrounds and/or videos which is what we currently can work with.

Changing it to "media" would need further discussion, as this was not what was originally proposed. I'll gather some more opinions on it.

Author: Resolved with the most recent update.

>   - Beatmaps using AI-generated backgrounds should instead credit them as such in the description.

Reviewer 2: This is super unclear to the point of WTF.

The credit should go in the description instead of somewhere else? Bad sentence IMO.

But ignoring that, it is ambiguous in its aim. Need more context to even make a call on whether I even agree with this being added.

Author: This was discussed in the meeting held on the 7th of October. It was catalyzed by a [recent discussion regarding how AI backgrounds should be credited](https://osu.ppy.sh/beatmapsets/2066427/discussion/-/generalAll#/3966141).

Currently, the aim of this change is to explicitly mention how AI Backgrounds should be credited moving on, and the importance of the term "generated" being used when doing so.

Other things such as website tags were discussed during the meeting, but extra efforts would require the game to take a stance for AI content, so for now we're only trying to change the way these are credited as otherwise there is no clause that makes mentions of this.

Please let me know if you need to look at the meeting notes where this was discussed and ping you accordingly!

Author: Resolved with the most recent update.

>   - Beatmaps using AI-generated backgrounds should instead credit them as such in the description.

Reviewer 2: If we're looking to make this as clear as day, I'd suggest something like this:

```suggestion
  - Beatmaps with AI-generated backgrounds should explicitly mention this in the description. This is to avoid any controversy from creators incorrectly claiming creative input (ie. claiming to have "illustrated" the background themselves).
```

Author: I'm keen on making the intent behind the clarification clearer.

Two things I'd change would be `controversy` to something less informal or "direct" as the purpose of crediting these works is also to be informative. Second thing would be changing `ie.` for `e.g.` which seems to be used in the RC when providing examples.

```suggestion
  - Beatmaps with AI-generated backgrounds should explicitly mention this in the description. This is to avoid creators from incorrectly claiming creative input (e.g. claiming to have "illustrated" the background themselves).
```

Reviewer 2: Fair points. I also thing it should be generalised to all imagery, not just backgrounds. How does this phrasing feel?

```suggestion
  - Beatmaps with AI-generated imagery should explicitly attribute it as being AI-generated, not "illustrated" or otherwise. This helps avoid any ambiguity over the creative input / process.
```

Author: When it comes to phrasing I'd probably not be the best person to ask haha.

Using "imagery" may blur the line a little, as we aren't referring to the files themselves but the contents of backgrounds/videos. Say for example, a background with AI assets or tweaks would potentially fall there, even if it wasn't fully generated by a model. Is this something we want to account for? 

Otherwise, we could use "media" or a similar variation of that as you pointed out before to refer to files in their entirety.

Will need more eyes on `This helps avoid any ambiguity over the creative input / process.` as I cannot judge how well it fits wording wise. But the idea feels good.

Reviewer 2: Imagery was intended to include storyboard etc.

Author: Hmm alright yeah fair. Not opposed to it if this is meant to encompass those kinds of assets.

Reviewer: the wording doesn't need to be opinionated or embellished beyond the main point of marking ai content as ai content. i don't get why a point about "creative input" needs to be made in the same breath

```suggestion
  - AI-generated media in beatmaps should explicitly be attributed as being AI-generated.
```

both the original beatmap thread and internal nat thread also seem to partly revolve around not wanting people to say ai images are "illustrated", although if they're credited as "ai-generated" anyway then is such a remark really necessary?

Reviewer 2: It provides reasoning. Something which is sorely missing from the guidelines at the moment.

Reviewer: well maybe something like this then

```suggestion
  - AI-generated media in beatmaps should explicitly be attributed as being AI-generated, not illustrated or otherwise. This is to prevent misrepresentation of the creation process involved.
```

basically the last suggestion here but more direct

Reviewer 2: Sure, I can agree on that one.

Author: Looks good!

  - `osu!` itself should never be used in the source field except in cases where the track was specifically created for osu! (also known as an [osu! original](/wiki/osu!_originals)), such as commissioned tournament tiebreaker tracks.
- **Metadata that exceeds the field's limits (81 characters) must be shortened.** Start by dropping additional markers and if this still is not sufficient, indicate that the title has been shortened using `...` in a sensible place.
- **If the artist or title fields were shortened to fit in field limits, the omitted information must be added to the tags.**

> - **The `featured artist` tag must only be added to tracks listed in the [Featured Artist catalogue](https://osu.ppy.sh/beatmaps/artists).** The tag is used to describe whether a track is licensed by osu!.

Reviewer 3: the wording here no longer requires adding the featured artist tag. it needs to say more to the effect of "the tag is required if [...] and allowed only if [...]" (but theres probably a better way to word that)

Author: If that is an issue, it could be worded similar to what was originally proposed like:
```
The `featured artist` tag must be added only if the tracks used are listed in the [Featured Artist catalogue](https://osu.ppy.sh/beatmaps/artists).** The tag is used to describe whether a track is licensed by osu!.
```

Moving the `only` ahead will make it clear that adding the tag is still a must, but only if that condition is met.

Will need to consult this with Wala however as they were the one to propose such wording.

Reviewer: woops, didn't see this

```suggestion
- **The `featured artist` tag must be used for tracks listed in the [Featured Artist catalogue](https://osu.ppy.sh/beatmaps/artists) and for such tracks only.** The tag is used to describe whether a track is licensed by osu!.
```

Author: Reads better on my end. Will wait for Clayton's OK before committing.

Reviewer 3: yeah wala's suggestion is good
